### DISCUSSION TITLE
QR code without quiet space

### DISCUSSION THREAD

**Discussion by cliff-lane** [micjahn/ZXing.Net#585] [Q&A]
Is there a way to disable the quiet space when generating a QR code? I am trying to write an app to generate QR codes in house rather than a subscription but they are coming out as the wrong size due to the quiet space.

**Comment by micjahn** [comment]
You can try it with the following option:

```
var writer = new BarcodeWriter
{
   Options = new QrCodeEncodingOptions
   {
      NoPadding = true
   }
};
```
